README.md

```diff
@@ -168,6 +168,16 @@ At the moment the only supported target in OSX is `_system`.
 
 `_blank` and `_self` targets are not yet implemented and are ignored silently. Pull requests and patches to get these to work are greatly appreciated.
 
+### iOS Quirks
+
+Since iOS 13 iPads try to adapt their content mode / user agent for the optimal browsing experience. This may result in iPads having their user agent set to `Macintosh`. You can change this with the `PreferredContentMode` preference in "config.xml".
```

Comment:
Can I suggest a slightly longer explanation?

Since iOS 13 forked into iPadOS 13, iPads try to adapt their content mode / user agent for the optimal browsing experience. This may result in iPads having their user agent set to `Macintosh` making it hard to detect them as mobile devices using user agent string sniffing. You can change this with the `PreferredContentMode` preference in "config.xml".

Reply:
```suggestion
Since the introduction of iPadOS 13, iPads try to adapt their content mode / user agent for the optimal browsing experience. This may result in iPads having their user agent set to Macintosh making it hard to detect them as mobile devices using user agent string sniffing. You can change this with the PreferredContentMode preference in "config.xml".
```

Sounds good to me. However I would try to avoid naming iPadOS an iOS fork.